source/robot_model/include/robot_model/Model.hpp

```diff
@@ -330,6 +330,12 @@ class Model {
    */
   const pinocchio::Model& get_pinocchio_model() const;
 
+  /**
+   * @brief Getter of the pinocchio data object tied to the model
+   * @return the pinocchio data object
+   */
+  const pinocchio::Data& get_pinocchio_data() const;
```

Comment:
Not sure about the const getter since operations using the pinocchio data will a lot of the time want to modify the object.

Reply:
I feel like if we go for a reference to pinocchio data we might as well do the same for the model. What do you think? (This is not something we need I think -- I'll be testing -- but we might as well give that option)

Reply:
Yep sounds sensible